openpype/lib/transcoding.py

```diff
@@ -226,10 +198,13 @@ def parse_oiio_xml_output(xml_string, logger=None):
     # Fix values with ampresand (lazy fix)
     # - ElementTree can't handle all escaped values with ampresand
```

Comment:
I think you are to harsh to ElementTree - it just wants valid xml

Reply:
> they are building the xml as a string

I though that this is the issue, hoped not.